From 8dc67890c93e094b503bd26394c556ea47ab8d31 Mon Sep 17 00:00:00 2001
From: Christopher Degawa <ccom@randomderp.com>
Date: Mon, 20 Dec 2021 12:31:13 -0600
Subject: [PATCH 1/3] third_party: set INSTALL variables as cache

this would give the option to install only shaderc if they already have
the third party repos in their system

Signed-off-by: Christopher Degawa <ccom@randomderp.com>
---
 third_party/CMakeLists.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/third_party/CMakeLists.txt b/third_party/CMakeLists.txt
index a01f31e..842c5d6 100644
--- a/third_party/CMakeLists.txt
+++ b/third_party/CMakeLists.txt
@@ -20,9 +20,9 @@ set(SHADERC_TINT_DIR "${SHADERC_THIRD_PARTY_ROOT_DIR}/tint" CACHE STRING
 set(SHADERC_ABSL_DIR "${SHADERC_THIRD_PARTY_ROOT_DIR}/abseil_cpp" CACHE STRING
   "Location of re2 source")
 
-set( SKIP_GLSLANG_INSTALL ${SHADERC_SKIP_INSTALL} )
-set( SKIP_SPIRV_TOOLS_INSTALL ${SHADERC_SKIP_INSTALL} )
-set( SKIP_GOOGLETEST_INSTALL ${SHADERC_SKIP_INSTALL} )
+set( SKIP_GLSLANG_INSTALL ${SHADERC_SKIP_INSTALL} CACHE BOOL "Skip glslang installation" )
+set( SKIP_SPIRV_TOOLS_INSTALL ${SHADERC_SKIP_INSTALL} CACHE BOOL "Skip spirv-tools installation" )
+set( SKIP_GOOGLETEST_INSTALL ${SHADERC_SKIP_INSTALL} CACHE BOOL "Skip googletest installation" )
 
 # Configure third party projects.
 if(${SHADERC_ENABLE_TESTS})
-- 
2.43.0

